src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -285,7 +276,7 @@ protected override MethodSymbol FindExplicitlyImplementedMethod(BindingDiagnosti
 
         protected override TypeSymbol ExplicitInterfaceType => _explicitInterfaceType;
 
-        protected override bool HasAnyBody => _hasAnyBody;
+        protected override bool HasAnyBody => flags.HasAnyBody;
```

Comment:
>protected override bool HasAnyBody => flags.HasAnyBody; [](http://example.com/codeflow?start=8&length=55)

It feels like this property should be moved to ```SourceMemberMethodSymbol``` and made non-virtual.

Reply:
Sure :)